src/chapters/errors-and-debugging/error-types.rst

```diff
@@ -0,0 +1,93 @@
+Error Types
+===========
+
+.. index::
+   single: error; syntax
+   single: error; type
+   single: error; runtime
+   single: object; built-in
+
+An **error type** is the classification that JavaScript uses internally to group errors based on their cause. In future lessons, we will come to understand that an error type is actually something called a **built-in object**. For now, however, understanding the different types of errors will help us become faster at debugging.
```

Comment:
Replace "come to understand" with "learn".

Remove 'internally' and 'however'.